projects/dev.yaml

```diff
@@ -52,6 +52,10 @@ paths:
                   options: '{{options.citoid}}'
                 - path: v1/lists.js
                   options: '{{options.lists}}'
+                - path: v1/css.yaml
+                  options:
+                    response_cache_control: '{{options.purged_cache_control}}'
```

Comment:
Since it's static content we might want some different cache-control here and overall I think. `purged_cache_control` sets the Varnish caching very large and client caching kinda small which might not be the best - if we update CSS we want the changes to get to the app quicker then in 2 weeks.

Reply:
@Pchelolo I basically want to use the cache-control header from PCS. Should I just leave it off here?

Reply:
Just remove it here, or maybe rename to something like `css_cache_control` and never set a value to it.

Reply:
Ok, removed this line on each, left the `host` line, though, since I want to tell RB to look this up from MCS.